src/pages/groups/[groupId].tsx

```diff
@@ -184,6 +195,12 @@ const BalancePage: NextPageWithUser<{
                   ))}
                 </div>
               </div>
+              {groupDetailQuery?.data?.createdAt && (
+                <div className="mt-8">
+                  <p className="font-semibold ">Group created</p>
+                  <p>{format(groupDetailQuery.data?.createdAt, 'yyyy-MM-dd')}</p>
```

Comment:
Good idea! A note: when i18n should be implemented the format should also be localed.

Reply:
@KMKoushik Isn't that a problem for all the dates right now, because I cant see that toLocaleDateString is used anywhere in the project. In that case probably it should be added to all the places where dates are returned?

Reply:
we can change for this and update others when we change those files, mostly we don't have to change cuz, it will be just month and date, for space constraint

Reply:
Fixed, I kept the format function as a wrapper just in case we want to change it like to MMM dd or something else.